application/libraries/PluginManager/AuthPluginBase.php

```diff
@@ -85,17 +85,19 @@ public function remoteControlLogin()
      * Set authentication result to success for the given user object.
      *
      * @param User $user
+     * @param \LimeSurvey\PluginManager\PluginEvent, current event if not set
      * @return AuthPluginBase
      */
-    public function setAuthSuccess(User $user)
+    public function setAuthSuccess(User $user, \LimeSurvey\PluginManager\PluginEvent $event = null)
```

Comment:
Can you add a `use` statement on top of file, so we don't have to write the full namespace? Thanks.

Reply:
Or namespace\ ?